src/main/java/me/roinujnosde/titansbattle/utils/DiscordWebhook.java

```diff
@@ -0,0 +1,391 @@
+package me.roinujnosde.titansbattle.utils;
+
+import javax.net.ssl.HttpsURLConnection;
+import java.awt.Color;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.reflect.Array;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Class used to execute Discord Webhooks with low effort
+ */
+public class DiscordWebhook {
+
+    private final String url;
+    private String content;
```

Comment:
Only those two fields are used in this class. 
The rest (and their respective methods) can be removed.
Also the other classes (Embed, etc).

Reply:
Done